Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -319,6 +319,11 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
             documentAttachments: content.documentAttachments,
             imageAttachments: content.imageAttachments
         )
+        
+        attachmentsVC.content = content.attachments.map {
+            ($0.payload as? AttachmentPreviewProvider) ?? DefaultAttachmentPreviewProvider()
```

Comment:
shouldn't the absence of payload be somehow logged at least as `info`?

Reply:
fixed, thanks!